gravitee-apim-console-webui/src/management/api/history-v4/deployment-info/deployment-info.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpTestingController } from '@angular/common/http/testing';
+import { ActivatedRoute } from '@angular/router';
+
+import { DeploymentInfoComponent } from './deployment-info.component';
+
+import { fakeEvent } from '../../../../entities/management-api-v2';
+import { CONSTANTS_TESTING, GioHttpTestingModule } from '../../../../shared/testing';
+
+describe('DeploymentInfoComponent', () => {
+  const API_ID = 'an-api-id';
+  const API_VERSION_ID = 'event-version-id';
+
+  let component: DeploymentInfoComponent;
+  let fixture: ComponentFixture<DeploymentInfoComponent>;
+  let httpTestingController: HttpTestingController;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [GioHttpTestingModule],
+      declarations: [DeploymentInfoComponent],
+      providers: [{ provide: ActivatedRoute, useValue: { snapshot: { params: { apiId: API_ID, apiVersionId: API_VERSION_ID } } } }],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(DeploymentInfoComponent);
+    component = fixture.componentInstance;
+    httpTestingController = TestBed.inject(HttpTestingController);
+    fixture.detectChanges();
+
+    expectApiEventRequest();
+  });
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+
+  it('should create component with proper page title', async () => {
+    fixture.detectChanges();
+    expect(component).toBeTruthy();
+    expect(getPageTitle()).toEqual('Version 1');
+  });
+
+  function expectApiEventRequest(response = fakeEvent()) {
+    const req = httpTestingController.expectOne(`${CONSTANTS_TESTING.env.v2BaseURL}/apis/${API_ID}/events/${API_VERSION_ID}`);
+    if (!req.cancelled) req.flush(response);
```

Comment:
if you have canceled request, maybe the component is making several calls when it shouldn't be, and you need to correct it rather than hide this in the tests?

Reply:
I suppose some mindless copy paste from my side. Fortunately, there was no redundant calls in this component so I just removed this `if (!req.cancelled)` part and it still passes tests